practice_5/task_2.js

```diff
@@ -0,0 +1,23 @@
+let mentor = { 
+    course: "JS fundamental", 
+    duration: 3,
+    direction: "web-development",
+    name: "Petro",
+    age: 35,
+};
+
+let showProps = (obj) => {
+    let propertyNames = Object.keys(obj);
```

Comment:
propertyNames is already array of keys
there is the same command for array of values

Reply:
Thanks, fixed